app/javascript/components/explore/StudyGeneField.js

```diff
@@ -114,7 +116,14 @@ export default function StudyGeneField({ genes, searchGenes, allGenes }) {
             isValidNewOption={() => false}
             noOptionsMessage={() => (inputText.length > 1 ? 'No matching genes' : 'Type to search...')}
             options={geneOptions}
-            onChange={value => setGeneArray(value ? value : [])}
+            onChange={value => {
+              // react-select doesn't expose the actual click events, so we deduce the kind
+              // of operation based on whether it lengthened or shortened the list
+              const newValue = value ? value : []
+              const actionName = value.length > geneArray.length ? 'add' : 'remove'
+              log('input:change', { text: `study-gene-search:${actionName}` })
```

Comment:
Blockers: 
* SCP naming pattern for interactions is `action:element`, not `element:action` (`change:menu`, `click:link`, etc.).  Maybe that's worth changing later, but for now let's go for internal consistency.
* The `text` value here is inconsistent with other uses of `text` in Mixpanel
* We should capture that A) this is a multiselect, and B) what value is being added or removed

Either suggestion below would satisfy those concerns.

Option A: Enable easier comparison, breakdowns, etc. across search types and contexts, like existing `search` event does:

```suggestion
              log('search-interaction', { 
                type: 'gene',
                scope: 'study', 
                action: `${actionName}:multiselect`
                text: value
              })
```

Option B: Enable easier comparison, breakdowns across all multiselect actions:

```suggestion
              log('$actionName:multiselect', { 
                widgetName: `study-gene-search`,
                action: actionName
                text: value
              })
```
```